content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
 
->Tip
+>टिप
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>यह मार्गदर्शिका ऐसे लोगों के लिए डिज़ाइन की गई है जो **कॉन्सेप्ट्स को चरण दर चरण** सीखना पसंद करते हैं। यदि आप ऐसा करके सीखना पसंद करते हैं, तो हमारे [व्यावहारिक ट्यूटोरियल](/tutorial/tutorial.html) को देखें। आपको यह मार्गदर्शिका और ट्यूटोरियल एक-दूसरे के पूरक लग सकते हैं।
 
-This is the first chapter in a step-by-step guide about main React concepts. You can find a list of all its chapters in the navigation sidebar. If you're reading this from a mobile device, you can access the navigation by pressing the button in the bottom right corner of your screen.
+मुख्य React कॉन्सेप्ट्स के बारे में चरण-दर-चरण मार्गदर्शिका में यह पहला अध्याय है। आप नेविगेशन साइडबार में इसके सभी अध्यायों की सूची पा सकते हैं। यदि आप इसे मोबाइल डिवाइस से पढ़ रहे हैं, तो आप अपनी स्क्रीन के निचले दाएं कोने में बटन दबाकर नेविगेशन का उपयोग कर सकते हैं।
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+इस मार्गदर्शिका का प्रत्येक अध्याय पहले के अध्यायों में प्रस्तुत ज्ञान पर आधारित है। आप वे साइडबार में दिखाई देने वाले “मेन कॉन्सेप्ट्स” गाइड चैप्टर्स को एक क्रम में पढ़कर अधिकांश React को सीख सकते हैं। उदाहरण के लिए, [“JSX का परिचय”](/docs/introducing-jsx.html) इस के बाद अगला अध्याय है।
 
-## Knowledge Level Assumptions {#knowledge-level-assumptions}
+## ज्ञान स्तर की मान्यताएँ {#knowledge-level-assumptions}
 
-React is a JavaScript library, and so we'll assume you have a basic understanding of the JavaScript language. **If you don't feel very confident, we recommend [going through a JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level** and enable you to follow along this guide without getting lost. It might take you between 30 minutes and an hour, but as a result you won't have to feel like you're learning both React and JavaScript at the same time.
+React एक जावास्क्रिप्ट लाइब्रेरी है, और इसलिए हम मान लेंगे कि आपको जावास्क्रिप्ट भाषा की बुनियादी समझ है। **यदि आप बहुत आत्मविश्वास महसूस नहीं करते हैं, तो हम अनुशंसा करते हैं कि आप अपने ज्ञान के स्तर की जांच करने के लिए एक [जावास्क्रिप्ट ट्यूटोरियल से गुजरें](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript)** ताकि यह आपको बिना खोए इस गाइड का अनुसरण करने में सक्षम करें।  यह आपको 30 मिनट और एक घंटे के बीच ले सकता है, लेकिन इसके परिणामस्वरूप आपको ऐसा महसूस करने की ज़रूरत नहीं है कि आप एक ही समय में React और जावास्क्रिप्ट दोनों सीख रहे हैं।
```

Comment:
हम मान लेंगे कि आपको जावास्क्रिप्ट -> hum ye maan kar chalenge ki aapko

Reply:
बहुत आत्मविश्वास महसूस नहीं -> in english it feels okay, but I feel you should add javascript main confident nai hain to in the hindi one

Reply:
लेकिन इसके परिणामस्वरूप आपको ऐसा महसूस करने की ज़रूरत नहीं है कि आप एक ही समय में React और जावास्क्रिप्ट दोनों सीख रहे हैं। -> grammar does not feel right